apps/juicebox_web/assets/js/reactions/video-frame.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component, PropTypes } from 'react';
+const frameAsset = require.context('../../images/frames', false);
```

Comment:
No idea really. Would `import * as frameAssets from '../../images/frames';` not work?